native/acter/src/api/verification.rs

```diff
@@ -720,7 +728,11 @@ async fn sas_verification_handler(
                         .emojis
                         .iter()
                         .map(|e| {
-                            let chr = e.symbol.chars().next().expect("first char not found");
+                            let chr = e
```

Comment:
same as above. I wonder if we don't want to put that into a singular location and only call off it... like have a `VerificatonEmoji::from(emoji)?` or something...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1252/commits/7d8760ae4ea5b2873de5d72ae7eed146743f318a